From b510dfef98a3de763c60df4fef6189ba5451eaf9 Mon Sep 17 00:00:00 2001
From: Alexander Gavrilov <angavrilov@gmail.com>
Date: Mon, 14 Jul 2008 22:56:47 +0400
Subject: [PATCH] Temporary workaround: use strtol instead of strtoll.

(Until somebody figures out how to compile it properly)

It breaks CVS commit 1.25 - may fail on Win95, etc.
---
 source/winsup/utils/kill.cc |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/source/winsup/utils/kill.cc b/source/winsup/utils/kill.cc
index e21646f..7446c59 100644
--- a/source/winsup/utils/kill.cc
+++ b/source/winsup/utils/kill.cc
@@ -241,7 +241,7 @@ main (int argc, char **argv)
 	case '?':
 	  if (gotasig)
 	    {
-	      pid = strtoll (argv[optind], &p, 10);
+	      pid = strtol (argv[optind], &p, 10);
 	      if (pid < 0)
 		goto out;
 	      usage ();
@@ -264,7 +264,7 @@ out:
   while (*argv != NULL)
     {
       if (!pid)
-	pid = strtoll (*argv, &p, 10);
+	pid = strtol (*argv, &p, 10);
       if (*p != '\0'
           || (!force && (pid < LONG_MIN || pid > LONG_MAX))
           || (force && (pid <= 0 || pid > ULONG_MAX))) 
-- 
1.5.6.2.1111.g806df

